alembic.rst

```diff
@@ -0,0 +1,84 @@
+Alembic
+=======
+Alembic is a database migration tool for usage with SQLAlchemy Database. The full documentation can be found here: 
+In invenio the alembic environment is already defined in invenio-db
+
+Creating new revisions
+----------------------
+
+When working with an invenio module that already has alembic revisions, after modifying a model we can create a new alembic revision with:
```

Comment:
It would be better if users could follow the script from top to bottom without jumping back and forth. Also we need to explain as if the user just discovered alembic and/or invenio-db and does not now how to search for alembic scripts.

What do you think instead of this:

===

Create an alembic upgrade script
========================

The following procedure will add an alembic upgrade script to invenio-unicorn for demonstration purpose.

First you need to check if your module has already some alembic scripts. Open the `setup.py` file and see if there is an `invenio_db.alembic entrypoints` property like this:
```python
setup(
    ...
    entry_points={
        ...
        'invenio_db.alembic': [
            'invenio_unicorn = invenio_unicorn:alembic',
        ]
...
})
```
If there is one, go the the package `invenio_unicorn.alembic` and check if there is a python script in it. It should have a name similar to: `843bc79c426f_create_invenio_unicorn_branch.py` (the numbers and letters at the beginning will not be the same).

If the `setup.py` file does not contain the `invenio_db.alembic` property or if the `invenio_unicorn.alembic` package is empty you need to create an upgrade branch first, continue to the next section. Otherwise you can skip the next section.

Create an upgrade branch
--------------------------

...

Create the new upgrade script
------------------------------

...